ui/src/components/Orchestration/MakeAccountOffer.tsx

```diff
@@ -0,0 +1,89 @@
+import { AgoricWalletConnection } from '@agoric/react-components';
+import { DynamicToastChild } from '../Tabs';
+import { useContractStore } from '../../store/contract';
+
+export const makeAccountOffer = async (
+  wallet: AgoricWalletConnection,
+  addNotification: (arg0: DynamicToastChild) => void,
+  selectedChain: string,
+  setLoadingCreateAccount: React.Dispatch<React.SetStateAction<boolean>>,
+  handleToggle: () => void,
+  setStatusText: React.Dispatch<React.SetStateAction<string>>
+) => {
+  if (!selectedChain) {
+    addNotification({
+      text: `Please Select Chain`,
+      status: 'error',
+    });
+    setLoadingCreateAccount(false);
+    handleToggle();
+    return;
+  }
+
+  const { instances } = useContractStore.getState();
```

Comment:
the name `instances` is a little surprising... it's `instance` in `agoricNames` (that's orthogonal to this PR, so IOU an issue)